packages/usa-checkbox/src/usa-checkbox--tile.twig

```diff
@@ -53,32 +53,5 @@
       >
       <label class="usa-checkbox__label" for="check-historical-carver-2">George Washington Carver</label>
     </div>
-    <div class="usa-checkbox">
-      <input
-        class="usa-checkbox__input usa-checkbox__input--tile"
-        id="check-historical-dubois-2"
-        type="checkbox"
-        name="historical-figures-2"
-        value="web-dubois"
-        data-indeterminate
-        {% if disabled_state == "disabled" %}disabled{% endif %} 
-        {% if disabled_state == "aria-disabled" %}aria-disabled="true"{% endif %}
-      >
-      <label class="usa-checkbox__label" for="check-historical-dubois-2">W.E.B. DuBois</label>
-    </div>
-    <div class="usa-checkbox">
-      <input
-        class="usa-checkbox__input usa-checkbox__input--tile"
-        id="check-historical-marshall-2"
-        type="checkbox"
-        name="historical-figures-2"
-        value="thurgood-marshall"
-        checked
-        data-indeterminate
-        {% if disabled_state == "disabled" %}disabled{% endif %} 
-        {% if disabled_state == "aria-disabled" %}aria-disabled="true"{% endif %}
-      >
-      <label class="usa-checkbox__label" for="check-historical-marshall-2">Thurgood Marshall</label>
-    </div>
```

Comment:
Deletion makes sense because this was added in the original work. Doesn't appear in current [develop](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/develop/?path=/story/components-form-inputs-checkbox--aria-disabled-tile&p=all).

Reply:
Resolved in 34b7f68.